tensorflow/compiler/jit/tests/auto_clustering_test.cc

```diff
@@ -21,22 +21,27 @@ namespace tensorflow {
 namespace {
 class AutoClusteringTestImpl : public AutoClusteringTest {
  protected:
-  Status RunAutoClusteringTest(absl::string_view key) {
+  // Test auto-clustering with ${key}.pbtxt
+  Status RunAutoClusteringTestWithPbtxt(absl::string_view key) {
     string file_name_without_extension =
         absl::StrCat(testing::TensorFlowSrcRoot(), "/compiler/jit/tests/", key);
 
-    return AutoClusteringTest::RunAutoClusteringTest(
+    return AutoClusteringTest::RunAutoClusteringTestWithPbtxt(
         absl::StrCat(file_name_without_extension, ".pbtxt"),
         absl::StrCat(file_name_without_extension, ".golden_summary"));
   }
-};
 
-Status BenchmarkHelper(absl::string_view key, benchmark::State& state) {
-  return BenchmarkMarkForCompilation(
-      absl::StrCat(testing::TensorFlowSrcRoot(), "/compiler/jit/tests/", key,
-                   ".pbtxt"),
-      state);
-}
+  // Decompress file ${key}.pbtxt.gz into ${key}.pbtxt
```

Comment:
This is no longer accurate, now that we're not using temporary files.

Reply:
Agh. My oversight. Let me update it.